### DISCUSSION TITLE
How do you test template changes?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#4106] [General]
I'm working on: https://github.com/codesandbox/codesandbox-client/pull/4105

and I changed the template path, but it doesn't seem like any way I try to start the app shows my changes. :-\

`yarn start:home` only shows the home page, and the `/s` route 404s

`yarn start` seems to use stale data. :-\

**Comment by CompuIves** [comment]
Hey @NullVoxPopuli! That's indeed weird. When you insert an `alert()` in the `src/index.js`, does it show when running `yarn start`?

I've taken a closer look at the templates, and I think that we moved where we keep that data to here: https://codesandbox.io/s/templates-endpoint-3drzs?file=/src/data/official.ts:87-132. Does that also include the Ember template?

**Reply by NullVoxPopuli** [reply]
Aside from looking for the word ember, I'm not sure what to look for in that file. What are all the 5 character strings?

**Reply by NullVoxPopuli** [reply]
Oh also, ember is an existing template. I was wanting to try changing the URL of the template

**Reply by CompuIves** [reply]
Those strings are the ids of the sandboxes, I'll add a comment to clarify that! What is the id of the ember template? You can get it from the URL.

**Reply by NullVoxPopuli** [reply]
it's a git reference? https://codesandbox.io/s/github/mike-north/ember-new-output
